lua/core/menu/devices.lua

```diff
@@ -95,7 +112,7 @@ end
 
 m.redraw = function()
   local y_offset = 0
-  if(4<m.pos) and m.section ~= "midi" then
+  if(4<m.pos) and not (m.section == "midi" and m.mode == "list") then
```

Comment:
also fixed a small bug by added this missing condition

it didn't get noticed thus far given the previous number of entries in the `DEVICES` sub-menu.